### ISSUE TITLE
Tracker ID coming in HTTP payload is wrong.

### ISSUE THREAD

**Issue by andrewferrier** [owntracks/ios#465]
I seem to have a newly-introduced issue on iOS with the tracker ID. I use OwnTracks on two iOS phones with different (set by me) tracker IDs. With a recent update to OwnTracks from the App Store (unsure exactly when but within last month) it appears that OwnTracks is now ignoring the value I set and sending its own (presumably auto-generated value). Each phone sends a different value, and even more mysteriously, if I backspace over my tracker ID, the one the phone is sending (which I can see on my backend server) appears in its place. So it appears that a new feature has been introduced to send a tracker ID automatically, but that is *always* being sent, even when supposedly I override it. 

Let me know if I can provide any more details. Current (broken) version of OwnTracks is 9.5.9.

Thanks for your hard work on OwnTracks!

**Comment by jpmens** [comment]
I cannot confirm this: I'm seeing my configured (manually entered) TID.

**Comment by ckrey** [comment]
@andrewferrier there was a change in the Settings UI. For all relevant parameters the effective value is displayed in _italics_.

If you do not specify a tracker id explicitly, the last two letters of the deviceId are used. The deviceId in HTTP mode is a auto generated value. This is what you see when you erase your input from the tracker id field.

But as far as I tested, the explicitly specified tracker id is shown in the payload.

You may want to export your configuration and send it to us for checking

**Comment by jpmens** [comment]
@andrewferrier if you do want to send us your configuration (without passwords of course), you can do so to `support@owntracks.org`.

**Comment by andrewferrier** [comment]
OK, I think I figured out what happened here. I *think* my configuration got temporarily reset, for reasons I cannot determine. I then fixed that by re-importing my configuration `.json`, but in the meantime OwnTracks had already sent (and failed, hence queued) an update with the wrong TID value (which my server-side rejects). Temporarily allowing a wrong TID allowed it through, and now OwnTracks is sending the correct TID.

So I think the interface is working - and I'm happy to close this. I think the thing that's confusing is how old updates are sent with the values that were in effect at the time, and that's not obvious. The only way I can see round this problem though would be to have a view which shows queued updates in detail, and I realise that's a non-trivial request. 

Thanks for the help - helped my narrow it down.
